docs/user-guide/tls-configuration.md

```diff
@@ -0,0 +1,90 @@
+# Customizing Native TLS
+
+Zowe's servers have built-in TLS support to enable HTTPS connections.
+
+This is the default, and an alternative to using AT-TLS which is [documented here](./configuring-at-tls-for-zowe-server).
+
+:::info Required role: security administrator
+:::
+
+## Server Parameters
+
+Each Zowe server can be customized either by defining attributes within the `zowe.network.server` object of the Zowe YAML configuration file. The same object can be put within an individual component's configuration, such as `components.zss.zowe.network.server` for ZSS, which will allow you to customize each component separate from others.
+
+Extensions are recommended to adhere to this configuration, but you must check with documentation for extensions to be sure.
+
+### IP Addresses
+
+Zowe's servers by default use the TCP IP address `0.0.0.0` which assigns the servers to be available on all network interfaces available to the jobs.
+
+If this default is not desired, you can either change it either within Zowe or by setting [TCPIP port assignment statements](./address-network-requirements#ip-addresses).
+
+To customize this within Zowe, define the parameter `zowe.network.server.listenAddresses`. For example, to have all Zowe servers use IP `1.2.3.4`, except for App Server which will use IP `2.3.4.5`, set the following in your Zowe YAML:
+
+```yaml
+zowe:
+  network:
+    server:
+      listenAddresses:
+      - 1.2.3.4
+components:
+  app-server:
+    zowe:
+      network:
+        server:
+          listenAddresses:
+          - 2.3.4.5
+```
+
+
+### TLS Versions
+
+By default, Zowe servers use TLSv1.3.
+
+To customize this, you can use the parameters `zowe.network.server.tls.minTls` and `zowe.network.server.tls.maxTls`. The following values are allowed:
+
+* TLSv1.2
+* TLSv1.3
+
+Zowe defaults to the following configuration:
+
+```yaml
+zowe:
+  network:
+    tls:
+      minTls: "TLSv1.2"
+      maxTls: "TLSv1.3"
```

Comment:
Is this still exclusive with attls being enabled?
It's for a separate issue, but we think it should be allowed to customize these values even with at-tls enabled. Although there are other restrictions (such as ICSF hardware keys). We are not assuming all onboarded services will be AT-TLS, it should be still possible to use Java TLS for these scenarios even with AT-TLS enabled in Zowe.

Reply:
The schema does have these as exclusive. Maybe the doc should make this clear?

It's a good point about a hybrid scenario. It's also difficult to make intuitive because people might think these settings would control AT-TLS when AT-TLS is enabled, but they're unrelated.

Reply:
I think so if we are not planning on removing this exclusion, although I think we should.